dotcom-rendering/src/components/FormatBoundary.tsx

```diff
@@ -20,6 +21,12 @@ const displayContents = css`
 export const FormatBoundary = ({ format, children }: Props) => {
 	const { darkModeAvailable = false } = useConfig();
 
+	const [isStorybook, setIsStorybook] = useState(false);
```

Comment:
Yes we devised this temporary measure to be able to test light and dark scheme variations in Storybook, as the `FormatBoundary` is unaware of the scheme.

Maybe this could be set as an optional prop?

Reply:
Discussed this and in the interest of keeping with the delivery mindset I’ve suggested we address this as a follow-up. The potential options I can think of are:
- optional prop only set explicitely in Storybook stories 
- value set from a context provider in `splitTheme`
- value set from the Story args in `splitTheme`